Core/Src/main.c

```diff
@@ -138,7 +139,9 @@ int main(void)
   defaultTaskHandle = osThreadNew(StartDefaultTask, NULL, &defaultTask_attributes);
 
   /* USER CODE BEGIN RTOS_THREADS */
-  /* add threads, ... */
+  osThreadId_t tempThread = osThreadNew(monitor_temp_msb, NULL, NULL); 
+  osThreadId_t imuThread = osThreadNew(monitor_knuckle_msb, NULL, NULL); 
+  osThreadId_t centralThread = osThreadNew(monitor_central_msb, NULL, NULL);

```

Comment:
These functions take inputs, so the next step is to add in the arguments to the thread call and to change the inputs of the function to reflect that. See Cerberus for examples

Reply:
Also make sure it compiles without warnings, which I can show you at the meeting